doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -286,6 +286,15 @@ Bluetooth samples
   * :ref:`bluetooth_central_hids`
   * :ref:`peripheral_hids_keyboard`
 
+* Updated the non-secure target ``nrf5340dk/nrf5340/cpuapp/ns`` configurations of the following samples to properly configure the TF-M profile instead of using the predefined minimal TF-M profile:
```

Comment:
```suggestion
* Updated the configurations of the non-secure ``nrf5340dk/nrf5340/cpuapp/ns`` board target in the following samples to properly use the TF-M profile instead of the predefined minimal TF-M profile:
```